vignettes/hook-arguments.Rmd

```diff
@@ -31,8 +31,16 @@ The following hooks take arguments:
   `base::grep()` to ignore further files from the hook. Argument `lang` is
   passed to `spelling::spell_check_files()`.
 
-* `lintr`: Arguments are not supported. Instead, `lintr` config should be
-  specified in a `.lintr` config file in Debian Control Field Format as
-  specified in the [`.lintr`
-  documentation](https://github.com/jimhester/lintr#project-configuration).
+* `lintr`: Argument `--warning_only` changes the behavior of the pre-commit to
+  be non-blocking. You should set this with the field verbose:true.
 
+  ```
+  id: lintr
+  args: [--warning_only]
+  vebose: True
+  ```
+  When configured this way, lintr prints lint errors as they appear. Other
+  arguments are not supported. Instead, `lintr` config should be specified in a
+  `.lintr` config file in Debian Control Field Format as specified in the
+  [`.lintr`
+  documentation](https://github.com/jimhester/lintr#project-configuration).
```

Comment:
Should we remove this line break here?

Reply:
It depends on how your formatter handles lines greater than 80 chars. Breaking on the space is one way to do it, and if I edit this again, this is likely to be reproduced.

I could fix this, but there isn't a really strong need for the effort. Please let me know.